### PR TITLE
Fix/RTENU-109 filter search functionality

### PR DESCRIPTION
**Pull request by NinaDang97**
- "Something went wrong" still exists. Turns out there's no null check for `content.sizeInBytes`
- Map mimetype before sending api request

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,9 +41,11 @@
     "concurrently": "^7.4.0",
     "constructs": "^10.0.0",
     "date-fns": "^2.29.3",
+    "lodash": "^4.17.21",
     "pretty-bytes": "^6.0.0"
   },
   "devDependencies": {
+    "@types/lodash": "^4.14.191",
```
Comment (mehtis) [inline package.json:48]: Lodash was added twice. Presumably not needed in devDependecies?

File: packages/frontend/src/pages/Search/SearchResult.tsx [hunk 2]
```diff
@@ -21,7 +22,7 @@ export const SearchResult = () => {
     term: query,
     from: formatYear(years[0]),
     to: formatYear(years[1]),
-    fileTypes: checkedList.mime,
+    fileTypes: checkedList.mime.map((mimeType: string) => mimeNamesMapping[mimeType as keyof typeof mimeNamesMapping]),
```
Comment (mehtis) [inline packages/frontend/src/pages/Search/SearchResult.tsx:25]: Curious: What does this bit do?

**Review by mehtis — CHANGES_REQUESTED** [review]
Lodash things needs to be fixed, other comment is just so that I understand what is happening.

File: package.json (hunk 1 above)
Comment (NinaDang97) [reply package.json:48]: Without "@types/lodash", Typescript can't find a declaration file for module 'lodash'. Otherway would be manually declare module lodash in types.d.ts but this faster

Comment (mehtis) [reply package.json:48]: Reading comprehension fail

**Review by mehtis — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (NinaDang97) [reply package.json:48]: Or I misunderstood your question

File: packages/frontend/src/pages/Search/SearchResult.tsx (hunk 2 above)
Comment (NinaDang97) [reply packages/frontend/src/pages/Search/SearchResult.tsx:25]: checkedList.mime contains array of checkbox names which are slightly different from the mime names

File: package.json (hunk 1 above)
Comment (mehtis) [reply package.json:48]: The fail was on me. Didn't notice the @types

**Pull request merged**